resources/farmnet-a/droplets.tf

```diff
@@ -0,0 +1,12 @@
+resource "digitalocean_droplet" "aries-farmnet-a" {
```

Comment:
Looks like this is a new instance we need for farmnet and not relaynet-related, right?

Reply:
yes